src/utils/vc/crypto/common/DockCryptoSignatureProof.js

```diff
@@ -133,7 +133,7 @@ export default withExtendedStaticProperties(
         c.status = credentialStatus;
       }
       return {
-        version: proof.version,
+        version: proof.presVersion,
```

Comment:
similar comment here, is this compatible with old credentials/presenting?

Reply:
Similar to above, can cause issue older presentations but not credentials. Fixed.